client/src/pages/ProfileDetails/ReviewForm/ReviewForm.tsx

```diff
@@ -1,26 +1,34 @@
-import { useState, useEffect, useContext } from 'react';
-import { Box, Typography, Card, CardMedia, CardContent, Button, TextField } from '@material-ui/core';
+import { useState, useContext } from 'react';
+import { Typography, Card, CardContent, Button, TextField } from '@material-ui/core';
 import Rating from '@material-ui/lab/Rating';
 import useStyles from './useStyles';
 import { createReview } from '../../../helpers/APICalls/createReview';
 import { useParams } from 'react-router-dom';
 import { AuthContext } from '../../../context/useAuthContext';
 import { SnackBarContext } from '../../../context/useSnackbarContext';
 
-const ReviewForm = (): JSX.Element => {
+const ReviewForm = (props: any): JSX.Element => {
```

Comment:
Please avoid using `any` as a type. Also, why don't we destructure the props? :)

Reply:
I was having issues setting the type. I will go back and fix this tomorrow if I have time.